tests/testfiles/tsmodule4.ts

```diff
@@ -0,0 +1,3 @@
+export const dummy: number = 42;
```

Comment:
nit: A typical name for this kind of directory is "fixtures".

Reply:
Good tip, updated